inst/extdata/support_severity.csv

```diff
@@ -1,6 +1,6 @@
 "Name","0 to 4","5 to 9","10 to 14","15 to 19","20 to 24","25 to 29","30 to 34","35 to 39","40 to 44","45 to 49","50 to 54","55 to 59","60 to 64","65 to 69","70 to 74","75 to 79","80 to 100"
 "p_C",0.25,0.26875,0.325,0.41875,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55,0.55
-"p_H",0.00657750963667825,0.00669393500309992,0.00170089091564484,0.000560809856252959,0.00074158594410974,0.00381910879736195,0.00521177604884915,0.00745166618557901,0.0119463511489046,0.0187894521844247,0.0239753692962161,0.0384705549936954,0.0879240088253233,0.102785507898414,0.216832234030657,0.386960282514078,0.392201739139098
+"p_H",0.019540650,0.008839994,0.008201863,0.010306579,0.020494920,0.022287780, 0.026423167, 0.026644751, 0.027651500, 0.029779858,0.056082939, 0.061260954, 0.090816417, 0.107041712, 0.184633309,0.222702109, 0.360564790
```

Comment:
This would end up being used by the vaccine_delay_fits task too - do we want that?

Reply:
I wouldn't know... would they be using fits by age for that paper? In case not, their parameters would need recalibrating with this new spline or I could put this p_H vector in `baseline.R`, what do you think?

Reply:
Probably best asking those leading on that paper. Otherwise I would suggest putting this version of the csv in the rtm_parameters_fits task

Reply:
I've done the latter in either case. The modified CSV is now in ncov